mill-universal-packager/src/io/github/hoangmaihuy/mill/packager/universal/UniversalPackagerModule.scala

```diff
@@ -58,4 +88,117 @@ trait UniversalPackagerModule extends PackagerModule {
     PathRef(T.dest)
```

Comment:
I think we have to clean `T.dest` first before copying. 
There is a case that I delete some files in `universalMappings` and they will still exist in `universalStage.dest`